src/bittytax/conv/parsers/coinbase.py

```diff
@@ -155,7 +155,9 @@ def _do_parse_coinbase(
             )
     elif row_dict["Transaction Type"] in (
         "Coinbase Earn",
+        "Inflation Reward",
```

Comment:
What is "Inflation Reward", is it from staking? Can you provide an example.

Reply:
These started showing up once I converted coins to ALGO. At the time, Coinbase would auto stake certain 
coins and pay these rewards.
https://coincodecap.com/coinbase-staking#:~:text=To%20stake%20any%20cryptocurrency%2C%20the,credits%20them%20into%20your%20wallet.
[reward_example.csv](https://github.com/BittyTax/BittyTax/files/13795947/reward_example.csv)